lib/ts/recipe/session/cookieAndHeaders.ts

```diff
@@ -167,7 +165,7 @@ export function setCookie(
     if (pathType === "refreshTokenPath") {
         path = config.refreshTokenPath.getAsStringDangerous();
     } else if (pathType === "accessTokenPath") {
-        path = "/";
+        path = config.accessTokenPath ? config.accessTokenPath.getAsStringDangerous() : "/";
```

Comment:
```suggestion
        path = config.accessTokenPath.getAsStringDangerous() === "" ? "/" : config.accessTokenPath.getAsStringDangerous();
```

Reply:
This change is suggested cause we will not make `accessTokenPath` optional in the normalised input type

Reply:
Done